src/test/scala/dotty/communitybuild/CommunityBuildTest.scala

```diff
@@ -74,4 +74,9 @@ class CommunityBuildTest extends BuildTest {
     gitUrl = "https://github.com/dotty-staging/pdbp",
     command = "compile"
   )
+
+  @Test def sourcecode = project(
+    gitUrl = "https://github.com/lihaoyi/sourcecode",
+    command = "++0.10.0 test"
```

Comment:
The community build will specify the scala version itself

Reply:
Fixed.